script/test-acceptance

```diff
@@ -28,4 +28,4 @@ cd -
 bundle .integration-daemon-stop
 
 # TODO: exit with $result status when tests are more stable.
-exit 0
```

Comment:
This exits the whole build process, should be `return`, missed that one previously :sweat:.